Library/Homebrew/cmd/deps.rb

```diff
@@ -52,45 +63,91 @@ def deps
       all?: ARGV.include?("--all"),
       topo_order?: ARGV.include?("-n"),
       union?: ARGV.include?("--union"),
+      for_each?: ARGV.include?("--for-each"),
     )
+    @with_reqs = ARGV.include?("--include-requirements")
+    @with_optional = ARGV.include?("--include-optional")
+    @with_build = ARGV.include?("--include-build")
+    @skip_recommended = ARGV.include?("--skip-recommended")
+    @annotate = ARGV.include?("--annotate")
```

Comment:
Feels weird to use class variables for some of `ARGV` and not others. I'd lean on the side of never using them.